Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -31,11 +76,22 @@ void OnSelect()
                 if (placing)
                 {
                     SpatialMappingManager.Instance.DrawVisualMeshes = true;
+
+                    Debug.Log(gameObject.name + " : " + "Removing existing world anchor if any.");
+
+                    // Remove existing world anchor when moving an object.
+                    DestroyImmediate(gameObject.GetComponent<WorldAnchor>());
+
+                    // Delete existing world anchor from anchor store when moving an object.                    
+                    anchorStore.Delete(SavedAnchorFriendlyName);
```

Comment:
Since this code is not tied to any events technically yes it possible if the user is quick enough. But I have never measured those ticks to know for sure. Do you have any recommendations?